js-exercises/user-check-password/user.js

```diff
@@ -0,0 +1,9 @@
+function User(username, password) {
+  Object.assign(this, { username, password });
```

Comment:
just curious why have you used Object.assign here? is it needed? I don't know myself. As far as I know we can just do this.username = username

Reply:
Both are almost similar, Object.assign is generally used in the place where we need to copy only the enumerable objects, but yes, in this place, this.username serves the purpose.